### ISSUE TITLE
directory tree consistency

### ISSUE THREAD

**Issue by davaya** [spdx/spec-v3-template#8]
The model file hierarchy has directories "Core" and "Software".

The Core directory has subdirectories "Classes", "Properties" and "Vocabularies" which each contain markdown files of their respective types.

The Software directory has no subdirectories, all the markdown files of the various types are mixed together, for example SoftwarePackage.md describes a Class but SoftwarePackageType.md describes a Vocabulary.

For consistency each of the top-level directories should have the same structure, with markdown files either mixed together or segregated by type.  (I believe separate directories for each type is more readable and reliable.)

For consistency each of the markdown files of each type should use the same markdown headings.  For example the "SoftwarePackageType" vocabulary uses the "Instances" heading, but the "RelationshipTypeVocab" vocabulary uses the "Entries" heading.

Pick a single set of headings for each type of markdown file.

**Comment by nirmal-suthar** [comment]
#3 may be relevant here

**Comment by zvr** [comment]
Right, we finalized on the structure as it is shown in the `Core` namespace.

Once we have the actual spec repo, we should use the correct data.
